.devcontainer/post-create.sh

```
@@ -68,24 +71,24 @@ retry 5 "prisma-data seed" pnpm --filter @klicker-uzh/prisma-data run seed:raw |
+  touch "$ROOT/apps/${app}/.env"
 done
 
 # Hatchet client token — automatically created by hatchet-lite-dev at /config/authdisabled-token.
 # The backend's HatchetClient.init runs at MODULE LOAD (not lazy), so the backend
 # CRASHES at boot without it — capture it here so post-start sources it before
 # turbo dev.
 echo "[post-create] Waiting for the Hatchet client token (/config/authdisabled-token)..."
-HATCHET_ENV=/workspaces/klicker-uzh/.devcontainer/.hatchet.env
+HATCHET_ENV="$ROOT/.devcontainer/.hatchet.env"
 : > "$HATCHET_ENV"
 for attempt in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)